CHANGELOG.md

```diff
@@ -2,6 +2,11 @@
 
 ## 1.2.0-beta.1 (Unreleased)
 
+### Breaking Changes
+
+### Bug Fixes
+
+- [[#1640]](https://github.com/Azure/azure-sdk-for-c/pull/1640) Update precondition on `az_iot_provisioning_client_parse_received_topic_and_payload()` to require topic and payload minimum size 1 instead of 0.
```

Comment:
```suggestion
- [[#1640]](https://github.com/Azure/azure-sdk-for-c/pull/1640) Update precondition on `az_iot_provisioning_client_parse_received_topic_and_payload()` to require topic and payload minimum size of 1 instead of 0.
```